src/ui/components/results/verticalresultscountcomponent.js

```diff
@@ -10,6 +11,8 @@ export default class VerticalResultsCountComponent extends Component {
         this.setState(verticalResults);
       }
     });
+
+    this._visibleForNoResults = (config.noResults || {}).visible;
```

Comment:
Minor nit, but let's make sure this is an actual boolean using '!!' or something like that. I think relying on truthy/falsy can get confusing, especially when a variable's name suggests it is a boolean.

Reply:
I'm still 50/50 about which I prefer so don't mind siding this way